app/helpers/mb_form_builder.rb

```diff
@@ -34,25 +34,20 @@ def mb_input_field(
   def mb_money_field(
     method,
     label_text,
-    type: "text",
+    type: :tel,
```

Comment:
Should this be `:tel`? Does that mean Telephone?

Reply:
yeah it's so that on mobile the number keyboard is pulled up!